app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -136,6 +136,30 @@ class SecureMessageController @Inject()(
         }
   }
 
+  def getContentDetail(id: String, contentType: MessageType): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I didn't get to review this PR, but my concern is that the same endpoint returns either a type of conversation or a type of letter, which are 2 distinct types with no common trait of any sort.

Reply:
good point @elabeca, on the call just now we've decided to have two different endpoints for these two types.